python/pyspark/ml/classification.py

```diff
@@ -857,6 +871,12 @@ def featureImportances(self):
         """
         return self._call_java("featureImportances")
 
+    @property
+    @since("2.0.0")
+    def trees(self):
```

Comment:
Oh, actually it caused by the PR updated. @holdenk removed the override ```trees``` methods at ```GBTClassifier.trees``` at commit 17cae03, and then my original comment moved to a wrong place. Whatever, we solved that issue now.